core/model/src/main/java/org/eclipse/rdf4j/model/util/Literals.java

```diff
@@ -9,10 +9,7 @@
 
 import java.math.BigDecimal;
 import java.math.BigInteger;
-import java.util.Date;
-import java.util.IllformedLocaleException;
-import java.util.Locale;
-import java.util.Objects;
+import java.util.*;
```

Comment:
I assume this is something that your editor has done automatically - could you change it? We don't use wildcards for imports.

If you are using Eclipse IDE, it's really easy to change this btw: go to 'Preferences' -> 'Java' -> 'Code Style' -> 'Organize imports', and where it says "number of imports needed for .*" set it to some very high number (99 for example).

Reply:
Yes, this is also done.